OpenIPSL/Data/PowerPlant/PFData/PF00030.mo

```diff
@@ -0,0 +1,25 @@
+within OpenIPSL.Data.PowerPlant.PFData;
+record PF00030 "Record for power flow solutions - Pload = 30 MW"
```

Comment:
The description string will be displayed in the drop-down list. The problem there is that since the string is quite long it gets cut short in the display (at least in Dymola, in OpenModelica that does not work yet anyway):
![Screenshot at 2023-03-28 09-54-23](https://user-images.githubusercontent.com/9332/228167756-0032a3a8-fecc-4c0b-b594-7d3f08f9ce2c.png)
So shortening it will improve this. 

E.g., 
![Screenshot at 2023-03-28 09-59-41](https://user-images.githubusercontent.com/9332/228169173-c3b61fbd-da72-4d35-baaf-237e9de1ce78.png)

Reply:
Done! Please see [commit b75a627](https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/b75a627df7e47556faa0385548a65b8dc4927139).

Reply:
Good but why are all those classes empty and don't set any values?

Reply:
I did not know what you mean with empty classes... Is it about the transformer classes? This structure is based on the PF records that Sergio Dorado proposed which includes records with pf solutions for transformers. The two examples in the Data package do not have any transformer, and therefore the transformer records are empty.

Reply:
OK I think the issue here is that you need to decide what should be part of new Data pakacge with useful data and what should be simply for demonstration purposes.  The former should not be some example but actually useable data. The latter should not be present in the Data package unless it is under the Examples. The reason is that people might start using classes from the Data package and rely on them "as is" and any future changes to them would then break their models. By definition, example models are not meant to be used in user models. At least not in a way that users can rely on them to be backwards compatible also in future releases. So try to be careful as to what is meant to be a framework one can rely on and what should just be a demonstration playground.

Reply:
Commits [1b9bdd9](https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/1b9bdd9e854ded929fc6af3e8276c8e4ea53a040) and [3a067f2](https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/3a067f26d68825e1738a86cf562dd534ec638066) aim to address this issue. Empty classes have been removed and the entire PFData package is now under the Examples package.